modules/atomic-widgets/prop-types/concerns/has-required-validation.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Elementor\Modules\AtomicWidgets\PropTypes\Concerns;
+
+use Elementor\Modules\AtomicWidgets\PropTypes\Contracts\Prop_Type;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * @mixin Prop_Type
+ * @mixin Has_Settings
+ */
+trait Has_Required_Validation {
```

Comment:
why not Has_Required?

Reply:
Changed to `Has_Required_Setting` is it better? `Has_Required` feels a little bit weird to me